lib/translation_maps/locations.rb

```diff
@@ -1,323 +1,325 @@
 {"anxa"=>"Forrestal Annex",
```

Comment:
Hmm...ok. Maybe a the code that's caching it could be wrapped in a Rake task, and this file added to .gitignore, just to avoid confusion?